commons/protos/policy_interface.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+message Graph {
+  message Node {
+    string version = 1;
+    string image = 2;
+    map<string, string> metadata = 3;
+  }
+
+  message Edge {
+    uint64 from = 1;
+    uint64 to = 2;
+  }
+
+  repeated Node nodes = 1;
+  repeated Edge edges = 2;
+}
+
+message PluginInput {
+  Graph graph = 1;
+  map<string, string> parameters = 2;
+}
+
+message PluginOutput {
+  message Error {
+    enum Status {
+      SUCCESS = 0;
+      FAILURE = 1;
+    };
+
+    Status plugin_status = 1;
+    string description = 2;
+    uint64 http_status = 3;
```

Comment:
I think this will go out-of-band if/when we use HTTP as a transport, not inside the protobuf.